tests/functional/elements.js

```diff
@@ -2,27 +2,234 @@
+    CONNECTION_BENCHMARK: 'VPNConnectionBenchmark',
+    CONNECTION_INFO_RETRY: 'connectionInfoErrorRetryButton',
+
+    // tips and tricks
+    TIPS_AND_TRICKS_VIEW: 'settingsTipsAndTricksPage',
+    TIPS_AND_TRICKS_POPUP_LOADER: 'tipsAndTricksIntroPopupLoader',
+    TIPS_AND_TRICKS_POPUP_CLOSE: 'tipsAndTricksIntroPopupCloseButton',
+    TIPS_AND_TRICKS_POPUP_DISCOVER: 'tipsAndTricksIntroPopupDiscoverNowButton',
+    TIPS_AND_TRICKS_POPUP_BACK: 'tipsAndTricksIntroPopupGoBackButton',
+
+    // tutorials
+    TUTORIAL_POPUP_PRIMARY_BUTTON: 'tutorialPopupPrimaryButton',
+    TUTORIAL_POPUP_SECONDARY_BUTTON: 'tutorialPopupSecondaryButton',
+    TUTORIAL_UI: 'tutorialUiRoot',
+    TUTORIAL_LEAVE: 'tutorialLeave',
+    TUTORIAL_LIST_HIGHLIGHT: 'layoutAll/columnAll/tutorialsSectionAll/tipsAndTricksSetionLoader/tutorialList/highlightedTutorial',
+
+}
+
+const initialView = {
+    SCREEN: 'initialStackView',
+    GET_HELP_LINK: 'getHelpLink',
+    GET_STARTED: 'getStarted',
+    LEARN_MORE_LINK: 'learnMoreLink',
+    SKIP_ONBOARDING: 'skipOnboarding',
+    ONBOARDING_NEXT: 'onboardingNext',
+    AUTHENTICATE_VIEW: 'authenticatingView',
+    CAP_PORTAL_BUTTON: 'captivePortalAlertActionButton',
+}
+
+const authView = {
+    EMAIL_INPUT: 'authStart-textInput',
+    EMAIL_INPUT_CONDITIOIN: 'authSignUp-passwordConditionEmailAddress',
+    START_BUTTON: 'authStart-button',
+    START_BACK_BUTTON: 'authStart-backButton',
+    SIGNIN_PASS_INPUT: 'authSignIn-passwordInput',
+    SIGNUP_PASS_INPUT: 'authSignUp-passwordInput',
+    PASS_INPUT_CONDITION: 'authSignUp-passwordConditionCommon',
+    PASS_LENGTH_CONDITION: 'authSignUp-passwordConditionLength',
+    SIGNIN_BUTTON: 'authSignIn-button',
+    SIGNUP_BUTTON: 'authSignUp-button',
+    SIGNIN_BACK: 'authSignIn-backButton',
+    SIGNUP_BACK: 'authSignUp-backButton',
+    SIGNUP_GET_HELP_LINK: 'authSignUp-getHelpLink',
+    SIGNIN_GET_HELP_LINK: 'authSignIn-getHelpLink',
+    ERROR_POPUP_BUTTON: 'authErrorPopup-button',
+    GET_HELP_LINK: 'authStart-getHelpLink',
+
+    verificationByEmailView: {
+        TEXT_INPUT: 'authVerificationSessionByEmailNeeded-textInput',
+        BACK: 'authVerificationSessionByEmailNeeded-backButton',
+        GET_HELP: 'authVerificationSessionByEmailNeeded-getHelpLink',
+        EMAIL_BUTTON: 'authVerificationSessionByEmailNeeded-button',
+    },
+        
+    verificationBySessionTotpView: {
+        GET_HELP_LINK: 'authVerificationSessionByTotpNeeded-getHelpLink',
+        BACK: 'authVerificationSessionByTotpNeeded-backButton',
+        TEXT_INPUT: 'authVerificationSessionByTotpNeeded-textInput',
+        TOTP_NEEDED_BUTTON: 'authVerificationSessionByTotpNeeded-button'
+    },
+
+    authUnblockCodeView: {
+        GET_HELP_LINK: 'authUnblockCodeNeeded-getHelpLink',
+        BACK: 'authUnblockCodeNeeded-backButton',
+        CODE_INPUT: 'authUnblockCodeNeeded-textInput',
+        CODE_BUTTON: 'authUnblockCodeNeeded-button',
+    }
+}
+
+const mobileOnBoardingView = {
```

Comment:
screen. Plus, do we want to match the name of the screens as in the `screens` folder?